Data/core_rulebook/cr_abilities_class.lua

```diff
@@ -73669,11 +73669,12 @@ DefineAbility({
   Name="Bard Class",
   Category="Internal",
   Add={
-    AddSpellCasterLevel({
```

Comment:
Hrm... It's sort of weird that it's `Add={{Kind="Language", Types={}}}` (an array of typed tables) and just just `Add={Language{Types={}}}` (a keyed dictionary of stuff to add).

Is the concern that a single ability might call "AddSpellCasterLevel" more than once?

Reply:
they are allowed to, I haven't seen any that can though.

Reply:
Hrm... I guess this is reasonable then?  Just leads to a sort of weird parser, but it's not the end of the world. :-)